pymc/model.py

```diff
@@ -363,9 +363,7 @@ def __init__(
         givens = []
         self._extra_vars_shared = {}
         for var, value in extra_vars_and_values.items():
-            shared = aesara.shared(
-                value, var.name + "_shared__", broadcastable=[s == 1 for s in value.shape]
-            )
+            shared = aesara.shared(value, var.name + "_shared__", shape=value.shape)
```

Comment:
We might not want to restrict the non-broadcastable shapes (we weren't before):

```suggestion
            shared = aesara.shared(value, var.name + "_shared__", shape=[1 if s == 1 else None for s in value.shape])
```